packages/synapse-react-client/src/components/CardGridWithLinks/CardGridWithLinks.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, Stack, Typography, Link, Card } from '@mui/material'
+import { Link as RouterLink } from 'react-router'
+
+type CardGridWithLinksConfig = {
+  title: string
+  description: string
+  link: string
+}
+
+export type CardGridWithLinksProps = {
+  linkText: string
+  cards: CardGridWithLinksConfig[]
+}
+
+const CardGridWithLinks = ({ cards, linkText }: CardGridWithLinksProps) => {
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        gap: 2,
+        width: '100%',
+        flexDirection: { xs: 'column', md: 'row' },
+      }}
+    >
+      {cards.map((card, index) => {
+        const encodedURI = encodeURI(card.link)
```

Comment:
Sorry, I was unclear. I meant: don't call `encodeURI` at all in this component. If the URI needs to be encoded, it should be encoded before passing it to `CardGridWithLinks`